pkg/cmd/cli/cli.go

```diff
@@ -36,9 +36,8 @@ run new-app:
   $ %[1]s new-app openshift/ruby-20-centos7~https://github.com/openshift/ruby-hello-world.git
 
 This will create an application based on the Docker image 'openshift/ruby-20-centos7' that builds
-the source code at 'github.com/openshift/ruby-hello-world.git'. To start the build, run
-
-  $ %[1]s start-build ruby-hello-world --follow
+the source code at 'github.com/openshift/ruby-hello-world.git'. Build and deployment will be
```

Comment:
Build will start automatically and deployment will start as soon as the build finishes. (deployment is waiting for an image to deploy ;-)